src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -16,6 +18,18 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+    switch (e.getActionCommand()) {
+      case "Figure Joe 1":
+          FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+          break;
+        case "Figure Joe 2":
+          FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+          break;
+          case "Figure Jane":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.